pkg/proxy/proxy.go

```diff
@@ -34,6 +34,12 @@ const (
 	ObotAuthProviderQueryParam = "obot-auth-provider"
 )
 
+type InvalidSessionErr struct{}
+
+func (InvalidSessionErr) Error() string {
+	return "invalid session"
+}
```

Comment:
I recently learned that you could also do `var InvalidSessionErr = errors.New("invalid session")` here and use `errors.Is(err, proxy.InvalidSessionErr)` in the other spot. Not sure if you think that would be cleaner.